cron/centreon-send-stats.php

```diff
@@ -30,6 +30,12 @@
 
 $db = $dependencyInjector['configuration_db'];
 
+function echoLog($message, $exception = null) {
+    $datetime = new DateTime();
+    $datetime->setTimezone(new DateTimeZone('UTC'));
+    $logEntry = is_null($exception) ? $message : $message . ' - ' . $exception;
```

Comment:
Do we want to display also stack trace ? If not, we should use $exception->getMessage()

Reply:
yes. in order to understand what happened